pkgs/top-level/ocaml-packages.nix

```
@@ -996,6 +992,8 @@ let
 
     parmap = callPackage ../development/ocaml-modules/parmap { };
 
+    ocaml_cairo = callPackage ../development/ocaml-modules/ocaml-cairo { };
```

```suggestion
    ocaml_cairo = callPackage ../development/ocaml-modules/ocaml-cairo { };
```
This is still out of order. As it is after `lwt`.